packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -726,6 +726,11 @@ class Editor implements KetcherEditor {
     KetcherLogger.log('Editor.redo(), end');
   }
 
+  public clearHistory() {
+    this.historyStack = [];
+    this.historyPtr = 0;
```

Comment:
Nitpick – it seems to be better implemented in History class and just invoked here as now it sort of exposes implementation details of History class here